src/Inventory/Home.js

```diff
@@ -46,34 +36,48 @@ export default class Home extends Component {
 						color={colors.tint}
 						size={24}
 						underlayColor='transparent'
-						onPress={() => navigation.navigate('CreateList')}
+						onPress={() => navigation.navigate('CreateList', { loadList: params.loadList})}
 						containerStyle={styles.rightButton}
 					/>
 				</View>
 			)
 		}
 	}
 
+	componentWillMount() {
+		this.props.navigation.setParams({ setModalVisible: this.setModalVisible });
```

Comment:
Why do we need this?

Reply:
It's a part of trying to navigate to the list right after creating it. It proved to me more difficult than I thought since I use the full modal for new list... But that can be a different task later. I'll just remove them.